Error when running storybook in projects

**Describe the bug**
Running `yarn storybook` in a fresh volto project throws error.
`Error: Cannot find module '@plone/volto/webpack-less-plugin'`

**To Reproduce**
Steps to reproduce the behavior:
1. create a volto project.
2. run `yarn storybook`

**Expected behavior**
Storybook should run and look for `.stories` in src/**

**Software:**

- OS: MacOs Monterey
- Browser Chrome
- Volto Version: `<latest>`

--- a/packages/generator-volto/CHANGELOG.md
+++ b/packages/generator-volto/CHANGELOG.md
@@ -4,6 +4,7 @@
 
 ### Added
 
+- Fix webpack-less-plugin import in storybook @nileshgulia1
 ### Changes
 
 ## 5.5.0 (2022-02-03)

--- a/packages/generator-volto/generators/app/templates/.storybook/main.js
+++ b/packages/generator-volto/generators/app/templates/.storybook/main.js
@@ -5,7 +5,7 @@ const fileLoaderFinder = makeLoaderFinder('file-loader');
 
 const projectRootPath = path.resolve('.');
 const createAddonsLoader = require('@plone/volto/create-addons-loader');
-const lessPlugin = require('@plone/volto/webpack-less-plugin');
+const lessPlugin = require('@plone/volto/webpack-plugins/webpack-less-plugin');
 
 const createConfig = require('../node_modules/razzle/config/createConfigAsync.js');
 const razzleConfig = require(path.join(projectRootPath, 'razzle.config.js'));

